lua/nvim-tree/view.lua

```diff
@@ -249,16 +254,24 @@ end
 local function grow()
   local starts_at = M.is_root_folder_visible(require("nvim-tree.core").get_cwd()) and 1 or 0
   local lines = vim.api.nvim_buf_get_lines(M.get_bufnr(), starts_at, -1, false)
-  -- 1 column of right-padding to indicate end of path
-  local padding = 3
+  -- number of columns of right-padding to indicate end of path
+  local padding = get_size(M.View.padding)
+
+  -- account for sign/number columns etc.
+  local wininfo = vim.fn.getwininfo(M.get_winnr())
+  if type(wininfo) == "table" and type(wininfo[1]) == "table" then
+    padding = padding + wininfo[1].textoff
+  end
+  print(padding)
```

Comment:
Please remove this stray debug `print` statement.
This gives me an extra "Press ENTER or type command to continue" prompt with the padding printed, whenever I toggle Nvim tree window.

Reply:
Yeah sorry that was my mistake, the problem was fixed a few days ago in master ^_^